samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -9,6 +9,13 @@
   <PropertyGroup>
     <RazorCompileOnBuild>true</RazorCompileOnBuild>
   </PropertyGroup>
+  
+  <ItemGroup>
+      <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+        <_Parameter1>Microsoft.AspNetCore.Mvc.AdditionalReference</_Parameter1>
+        <_Parameter2>MvcSandbox.PrecompiledViews.dll,false</_Parameter2>
```

Comment:
Can we treat this as an assembly name (short) instead of a filename? 

This could be used to load stuff from anywhere on the system since it's passed to path.combine. That seems like a bad hook to give. I'm not sure that there's really a security problem.  It's probably also a good idea to harden the code that loads the assembly, and have it make sure that the assembly comes from the *bin* directory, and otherwise ignore it.

Reply:
I'm following what we were doing on ViewsFeatureProvider, I'm totally cool with changing how this works, can we file a bug for it and change it on preview2? I want to get this done